docs/docs/spark-queries.md

```diff
@@ -301,6 +301,16 @@ SELECT * FROM prod.db.table.entries;
 | -- | -- | -- | -- | -- | -- |
 | 2 | 57897183625154 | 0 | 0 | {"content":0,"file_path":"s3:/.../table/data/00047-25-833044d0-127b-415c-b874-038a4f978c29-00612.parquet","file_format":"PARQUET","spec_id":0,"record_count":15,"file_size_in_bytes":473,"column_sizes":{1:103},"value_counts":{1:15},"null_value_counts":{1:0},"nan_value_counts":{},"lower_bounds":{1:},"upper_bounds":{1:},"key_metadata":null,"split_offsets":[4],"equality_ids":null,"sort_order_id":0} | {"c1":{"column_size":103,"value_count":15,"null_value_count":0,"nan_value_count":null,"lower_bound":1,"upper_bound":3}} |
 
+Note:
+
+1. The columns of the `entries` table correspond to the fields of the `manifest_entry` struct (see the [manifest file schema](../../spec.md#manifests) for the full definition):
+    - `status`: Used to track additions and deletions
+    - `snapshot_id`: The ID of the snapshot in which the file was added or removed
+    - `sequence_number`: Used for ordering changes across snapshots
+    - `file_sequence_number`: Indicates when the file was added
+    - `data_file`: Metadata about the data file
```

Comment:
Can you also add description of this struct?

Reply:
Sure, since `data_file` has 24 fields defined in https://iceberg.apache.org/spec/#manifests , can I improve the inline description and provide a link to the schema? It would look like this:

> - `data_file`: A struct with metadata, the fields are also defined in the [manifest file schema](../../spec.md#manifests)

Reply:
Yeah, but I can't find a link to the `data_file` definition.

Reply:
> Sure, since `data_file` has 24 fields defined in https://iceberg.apache.org/spec/#manifests , can I improve the inline description and provide a link to the schema? It would look like this:
> 
> > * `data_file`: A struct with metadata, the fields are also defined in the [manifest file schema](../../spec.md#manifests)

This makes sense to me, +1

Reply:
I don't think that's too much of an issue. We can link to the manifest right now and update the link later when we have proper anchors

Reply:
I've updated the description for `data_file` and added the link:

<img width="734" alt="data_file" src="https://github.com/user-attachments/assets/37316c95-f0e0-4480-9e6d-45045bedfb44" />

I will follow-up with a separate PR to add anchors after this one has been merged.